ansible_bender/cli.py

```diff
@@ -347,7 +347,11 @@ def _list_builds(self):
     def _get_logs(self):
         build_id = self.args.BUILD_ID
         log_lines = self.app.get_logs(build_id=build_id)
-        print("\n".join(log_lines))
+        if len(logs_lines) is not 0:
+            print("\n".join(log_lines))
+        else:
+            print("Sorry, the latest build has not got any log lines")
+            self.parser.print_help() 
```

Comment:
I'd drop this line - no need to print help when a build doesn't have any logs (which I think is almost impossible to happen :)